buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - ./gradlew build packageLambda
+
+artifacts:
+  files:
+    - 'frontend-service/build/distributions/frontend-service.zip'
```

Comment:
We don't actually vend/use these anywhere, don't think we need to actually archive them.

Reply:
I can remove it for now. I initially idea is we have S3 bucket as our artifact repo and used for further deployment, so we don't have to re-build and reload to different S3 bucket.